NorthstarDLL/logging/logging.cpp

```diff
@@ -220,3 +222,94 @@ void NS::log::FlushLoggers()
 
 	spdlog::default_logger()->flush();
 }
+
+// Wine specific functions
+typedef const char *(CDECL *wine_get_host_version_type)(const char**, const char**);
+wine_get_host_version_type wine_get_host_version;
+
+typedef const char *(CDECL *wine_get_build_id_type)(void);
+wine_get_build_id_type wine_get_build_id;
+
+typedef char *(CDECL *wine_get_unix_file_name_type)(LPCWSTR);
```

Comment:
These are leftovers from a previous attempt of this that need to be removed.
In a default Wine environment its possible to directly read files, from the host fs simply by translating the paths over.

Proton runs in a container with a static user `steamuser`, which means it cannot read user files unless specifically exposed by the container setup.

Reply:
I guess this got removed?